packages/react-native-hms/android/src/main/java/com/reactnativehmssdk/HMSRNSDK.kt

```diff
@@ -1630,7 +1630,13 @@ class HMSRNSDK(
           data.putString("metaData", null)
         }
         "peerCount" -> {
-          data.putInt("peerCount", hmsRoom.peerCount)
+          hmsRoom.peerCount.let {
```

Comment:
@stanwolverine  should the optional check be added for peerCount? Like so - `hmsRoom.peerCount?.let`

Reply:
It's not necessary, If the value is null, then we are using `putNull` method. I have tested this.